orchagent/intfsorch.cpp

```diff
@@ -195,8 +195,14 @@ bool IntfsOrch::setIntf(const string& alias, sai_object_id_t vrf_id, const IpPre
         return false;
     }
 
+    /* Don't add subnet route for IP addresses with maximum prefix length.
+     * It is a duplicate of IP2Me route */
+    if ((ip_prefix->isV4() && ip_prefix->getMaskLength() != 32) ||
```

Comment:
Just thinking, what is the use-case of configuring IPv4 address with /32 mask?

Reply:
Besides setting it on loopback interfaces, I don't know other use-case either. But Linux kernel does allow user to configure a /32 IPv4 address on netdevice.

Reply:
Do we want to allow user on this scenario?